### ISSUE TITLE
Add `nuzzle` namespace to top-level config keys that are specified by Nuzzle

### ISSUE THREAD

**Issue by stelcodes** [stelcodes/nuzzle#6]
Instead of using `:overlay-dir` as a top-level config key, use `:nuzzle/overlay-dir` instead. Repeat for all config settings specified by Nuzzle. This change goes with #5 because once the `:site-data` is merged into the top-level map, keyword keys specified by Nuzzle will potentially clash with keyword keys specified by the users. Adding the Nuzzle namespace will prevent breaking changes when adding new Nuzzle config keys. It also serves as a visual representation of what keys are Nuzzle settings and what keys aren't.

**Comment by stelcodes** [comment]
Done
